mirrord/cli/src/config.rs

```diff
@@ -153,10 +153,6 @@ pub(super) struct ExecArgs {
     #[arg(long = "steal")]
     pub tcp_steal: bool,
 
-    /// Pause target container while running.
-    #[arg(short, long, alias = "paws")]
```

Comment:
The hell was this alias? :laughing:

Reply:
I think @meowjesty is the genius behind this